.github/workflows/release-winget.yaml

```diff
@@ -7,26 +7,29 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-    - name: Update winget repository
-      uses: mjcheetham/update-winget@v1.0
+    - id: update-winget
+      name: Update winget repository
+      uses: mjcheetham/update-winget@v1.2
       with:
-        token: ${{ secrets.WINGET_TOKEN }}
-        repo: microsoft/winget-pkgs
```

Comment:
I think we still want `repo: microsoft/winget-pkgs`, right? Or, is that built-in to `v1.2`?

Reply:
It's the [default](https://github.com/mjcheetham/update-winget/blob/bbac470a681ce6fbd56c54b25238dfa15f463d53/action.yml#L50-L53)! However, we can keep if it's better to be explicit.

Reply:
Thanks! Using the default works for me.

Reply:
@APACGAMONDE